apps/bfd-server/bfd-server-docs/src/main/java/gov/cms/bfd/datadictionary/model/FhirMapping.java

```diff
@@ -0,0 +1,36 @@
+package gov.cms.bfd.datadictionary.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/** Data class for serializing FHIR mappings from data dictionary resource files. */
+@Setter
+@Getter
+public class FhirMapping {
+  /** Version. */
+  private String version;
+
+  /** Resource. */
+  private String resource;
+
+  /** Element. */
+  private String element;
+
+  /** FHIR Path. */
+  private String fhirPath;
+
+  /** String[] of discriminators. */
+  private String[] discriminator;
+
+  /** String[] of additional info. */
+  private String[] additional;
```

Comment:
Can we use lists?  I avoid using arrays as much as possible unless they are required for performance or some library requires them.

Reply:
Bob the Builder: "Yes we can."